docs/rest_api.rst

```diff
@@ -663,3 +663,51 @@ be rejected.
   :statuscode 404: If the channel does not exist
   :statuscode 409: If the given block number argument is invalid
   :statuscode 500: Internal Raiden node error
+  
+
+.. http:get:: /api/1/channels/history/(token_address)/(partner_address)
```

Comment:
This is just the example for a given partner address. I'd add a separate point for just the token_address